src/agent/debuglet.js

```diff
@@ -146,8 +146,10 @@ Debuglet.normalizeConfig_ = function(config) {
   var envConfig = {
     logLevel: process.env.GCLOUD_DEBUG_LOGLEVEL,
     serviceContext: {
-      service: process.env.GAE_SERVICE || process.env.GAE_MODULE_NAME,
-      version: process.env.GAE_VERSION || process.env.GAE_MODULE_VERSION,
+      service: process.env.GAE_SERVICE || process.env.GAE_MODULE_NAME ||
+        process.env.FUNCTION_NAME,
+      version: process.env.GAE_VERSION || process.env.GAE_MODULE_VERSION ||
+        'unversioned',
```

Comment:
When the app is running locally, we will be providing a serviceContext with `service` set to undefined and `version` set to `unversioned`. How would this display in the UI?